src/plugins/telemetry/public/services/telemetry_service.ts

```diff
@@ -126,6 +131,10 @@ export class TelemetryService {
     return this.isOptedIn;
   };
 
+  public canSendTelemetry = (): boolean => {
+    return Boolean(this.getIsOptedIn() && !this.isScreenshotMode);
```

Comment:
NITs:
1. Why do we need to wrap it as Boolean?
2. Can we check `isScreenshotMode` first to avoid running the additional checks of `getIsOptedIn()`?

Reply:
> Why do we need to wrap it as Boolean?

I wanted to be explicit about the return type and `getIsOptedIn` returns a `boolean | null`. Happy to update that function return value with a `Boolean()` instead?

> Can we check isScreenshotMode first to avoid running the additional checks of getIsOptedIn()?

Can do!